Data/MolStandardize/normalizations.txt

```diff
@@ -3,7 +3,7 @@
 Nitro to N+(O-)=O	[N,P,As,Sb;X3:1](=[O,S,Se,Te:2])=[O,S,Se,Te:3]>>[*+1:1]([*-1:2])=[*:3]
 Sulfone to S(=O)(=O)	[S+2:1]([O-:2])([O-:3])>>[S+0:1](=[O-0:2])(=[O-0:3])
 Pyridine oxide to n+O-	[n:1]=[O:2]>>[n+:1][O-:2]
```

Comment:
This matches more than just pyridines doesn't it?

Reply:
Yeah. These were all copied over from MolVS itself when Susan did the port. Here's the original source: https://github.com/mcs07/MolVS/blob/master/molvs/normalize.py#L50